code/index.html

```diff
@@ -4,9 +4,79 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
     <title>Project 4: Pizzabot</title>
   </head>
+
   <body>
-    <h1>This is your Pizza Bot</h1>
+    <main>
+      <!-- Form section-->
+      <section class="form-container">
+        <!-- Welcome message-->
+        <h1>Welcome to Evelina & Agnes pizza delivery service!</h1>
+        <div class="welcome-message-container">
+          <p id="welcomeMessage"></p>
+        </div>
+
+        <!-- Form-->
+        <form onsubmit="handleOrder(event)">
+          <!-- Ask what pizza they would like to order -->
+          <div class="question-container">
+            <p id="pizzaName"></p>
+
+            <!-- Input form for what pizza they would like to order -->
+            <input
+              type="text"
+              id="pizzaNameInput"
+              name="pizza"
+              placeholder="Name of pizzas goes here"
+            />
+          </div>
+
+          <!-- Ask how many pizzas they would like to order-->
+          <div class="question-container">
+            <p id="pizzaQuantity"></p>
+
+            <!-- Input form for ask how many pizzas-->
+            <input
+              type="number"
+              id="pizzaQuantityInput"
+              name="pizzanumber"
+              placeholder="Number of pizzas goes here"
+              min="1"
+              max="100"
```

Comment:
I'm glad you remembered about `min` attribute, so there is no option to order -2 pizzas!